pkg/kubelet/cm/topologymanager/topologymanager.go

```diff
@@ -114,15 +118,22 @@ func (m *manager) AddHintProvider(h HintProvider) {
  	m.hintProviders = append(m.hintProviders, h)
 }
 
-func (m *manager) RemovePod (podName string) {
- 	glog.Infof("Remove pod func")
+func (m *manager) AddPod(pod *v1.Pod, containerID string) error {
+	m.podMap[containerID] = string(pod.UID)
+	return nil
+}
+
+func (m *manager) RemovePod (containerID string) error {
+ 	podUIDString := m.podMap[containerID]
+	delete(m.podTopologyHints, podUIDString)
+	glog.Infof("[topologymanager] RemovePod - Container ID: %v podTopologyHints: %v", containerID, m.podTopologyHints)

```

Comment:
Could probably delete the entry for that containerID in the podMap too - reduce the entries

Reply:
Good idea, will do.